docs/ops/convolution/ConvolutionBackpropData_1.md

```diff
@@ -4,15 +4,15 @@
 
 **Category**: Convolution
 
-**Short description**: Computes the gradients of a Convolution operation with respect to the input. Also known as a Deconvolution or a Transposed Convolution.
+**Short description**: Computes 1D, 2D or 3D *ConvolutionBackpropData* operation with respect to the input and kernel tensors. Also known as a Transposed Convolution.
 
 **Detailed description**:
 
 ConvolutionBackpropData takes the input tensor, weights tensor and output shape and computes the output tensor of a given shape. The shape of the output can be specified as an input 1D integer tensor explicitly or determined by other attributes implicitly. If output shape is specified as an explicit input, shape of the output exactly matches the specified size and required amount of padding is computed.
```

Comment:
How about adding some reference article about this operation as other related convolution operation do? E.g. https://arxiv.org/abs/1603.07285

Reply:
As far as I remember we shall limit linking to external resources. @lazarevevgeny shall we add link for detailed convolution backprop description?

Reply:
I would say only to unofficial external resources, but not to official research publications/papers. Please have a look to other convolution operations spec:

- [Convolution](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/convolution/Convolution_1.md)
- [BinaryConvolution](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/convolution/BinaryConvolution_1.md)
- [GroupConvolution](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/convolution/GroupConvolution_1.md)
- [DeformableConvolution](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/convolution/DeformableConvolution_1.md)

Reply:
Ok, I've added link to arxiv article.